media/js/firefox/whatsnew/whatsnew-98-turningred-en.js

```diff
@@ -0,0 +1,89 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+(function () {
+    'use strict';
+
+    // https://davidwalsh.name/javascript-debounce-function
+    function debounce(func, wait, immediate) {
+        var timeout;
+        return function () {
+            var context = this,
+                args = arguments;
+            var later = function () {
+                timeout = null;
+                if (!immediate) {
+                    func.apply(context, args);
+                }
+            };
+            var callNow = immediate && !timeout;
+            clearTimeout(timeout);
+            timeout = setTimeout(later, wait);
+            if (callNow) {
+                func.apply(context, args);
+            }
+        };
+    }
+
+    function isInViewport(element) {
+        var bounding = element.getBoundingClientRect();
+        var elementHeight = element.offsetHeight;
+        var elementWidth = element.offsetWidth;
+
+        if (
+            bounding.top >= -elementHeight &&
+            bounding.left >= -elementWidth &&
+            bounding.right <=
+                (window.innerWidth || document.documentElement.clientWidth) +
+                    elementWidth &&
+            bounding.bottom <=
+                (window.innerHeight || document.documentElement.clientHeight) +
+                    elementHeight
+        ) {
+            return true;
+        } else {
+            return;
+        }
+    }
+
+    function onLoad() {
+        // Check if user prefers reduced motion.
+        var matchMediaNoMotion = window.matchMedia(
+            '(prefers-reduced-motion: reduce)'
+        ).matches;
+
+        var things = document.querySelector('.c-common-things');
+
+        if (matchMediaNoMotion) {
```

Comment:
**suggestion (non-blocking):** hide gradient or include the hide-gradient class as default in html to avoid unnecessary fade between the list and release notes when user prefers reduced motion.
```suggestion
        if (matchMediaNoMotion) {
            things.classList.add('hide-gradient');
```
<img width="618" alt="Screen Shot 2022-03-03 at 12 02 07 PM" src="https://user-images.githubusercontent.com/19650432/156562268-b027a7bc-be82-4779-a93e-785699415513.png">